src/app/App.tsx

```diff
@@ -1,10 +1,19 @@
+import { AppLocale } from '../context/locale/appLocale.enum';
+import { useLocale } from '../hooks/useLocale';
+
 export const App = () => {
+  let { formatMessage, locale, setLocale } = useLocale();
```

Comment:
Staramy się unikać `let`a. `let` pozwala na zmianę wartości. A dobrą praktyką jest, żeby wszystkie dane były niezmienialne, ponieważ może to powodować side effecty w różnych miejscach